tools/dm/data-synchronization-features.md

```diff
@@ -0,0 +1,454 @@
+---
+title: 数据同步功能
+summary: DM 提供的功能及其配置介绍
+category: tools
+---
+
+# 数据同步功能
+
+这个文档详细的介绍了 DM 提供的数据同步功能，以及相关的配置选项
+
+## Table routing
```

Comment:
seems some docs translated this (like <https://github.com/pingcap/docs-cn/blob/da69e9ee5f894f6cad0b36e244a50329414e6ae6/tools/dm/shard-merge-scenario.md>.
should we need to unify them.

Reply:
Use proper nouns, we can use `table routing`